src/main/java/mate/academy/bookshop/repository/impl/BookRepositoryImpl.java

```diff
@@ -48,4 +45,16 @@ public List<Book> findAll() {
             throw new RuntimeException("Can't find all books", e);
         }
     }
+
+    @Override
+    public Optional<Book> findById(Long id) {
+        try (Session session = sessionFactory.openSession()) {
+            return session.createQuery("FROM Book b WHERE b.id = :id", Book.class)
+                    .setParameter("id", id)
+                    .uniqueResultOptional();
```

Comment:
I am sure there is a better method like `find()` or something like that to not use a `createQuery()`